cameo/strain_design/deterministic/flux_variability_based.py

```diff
@@ -369,8 +440,8 @@ def run(self, surface_only=True, improvements_only=True, progress=True, view=Non
             ] = True
 
             df.loc[
-                ((df.lower_bound <= 0) & (df.lower_bound > 0)) | (
-                    (ref_lower_bound >= 0) & (df.upper_bound <= 0)),
+                (zero_overlap_mask & (df.lower_bound > 0)) | (
```

Comment:
How reliable is this in terms of numerical issues? Should this maybe use `non_zero_flux_threshold` or similar?

Reply:
I mean something like `zero_overlap_mask & (df.lower_bound > non_zero_flux_threshold)`

Reply:
What about modifying all solutions as in the following:
```py
df[df[["lower_bound", "upper_bound"]].abs() < non_zero_flux_threshold] = 0
```

Reply:
fine with me